CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Service Broker API spec 2.15 and above provide `organization_name` and `space_name`.
+  If these are available, they are added as annotations on the organization and space policies
+  that are created in Conjur.
+  [cyberark/conjur-service-broker#238](https://github.com/cyberark/conjur-service-broker/issues/238)
+
+
```

Comment:
Not a big deal, but if you need to make a change for another reason, please remove the extra line.

Reply:
Actually not a nit, it causes a failure with the parse-changelog.sh script.
ERROR: line 14, column 1: Expected one of '### ', '## ', '[Unreleased]: ' at line 14, column 1 (byte 590) 
So we do need that line removed.